modules/rank-eval/src/test/java/org/elasticsearch/index/rankeval/RatedRequestsTests.java

```diff
@@ -122,6 +122,7 @@ public void testXContentRoundtrip() throws IOException {
         }
```

Comment:
Note: the is almost identical to https://github.com/elastic/elasticsearch/pull/104684, which didn't get merged, so I'm uncertain whether I should merge this.